doc/source/_templates/layout.html

```diff
@@ -52,3 +39,17 @@
 {%- endif %}
 {{ super() }}
 {% endblock %}
+
+{%- block footer %}
+{{ super() }}
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script type="text/javascript"> docsearch({
+apiKey: 'c8367ae0f87be4fd19cb01bae97ef4f2',
+indexName: 'numpy',
+inputSelector: 'form input[type=text]',
```

Comment:
This seems dangerously vague - can we use a more specific filter?
```suggestion
inputSelector: '#algolia-searchbar form input[type=text]',
```

Reply:
Agreed.